share/spice/ruby_gems/ruby_gems.js

```diff
@@ -26,13 +26,42 @@
             templates:{
                 group: 'text',
                 detail: false,
-		item_detail: false
+                item_detail: false,
+                variants: {
+                    tile: 'basic1',
+                    tileTitle: '1line',
+                    tileFooter: '2line',
+                    tileSnippet: 'large'
+                },
+                options: {
+                    footer: Spice.ruby_gems.footer
+                }
+            },
+            sort_fields: {
+                downloads: function(a, b) {
+                    return a.downloads > b.downloads ? -1 : 1;
+                }
             },
+            sort_default: 'downloads',
             normalize : function(item){
+                var licenses = [];
+
+                if (item.licenses) {
+                    var keys = Object.keys(item.licenses);
+
+                    for (var i = 0; i < keys.length; i++) {
+                        licenses.push(item.licenses[keys[i]]);
+                    };
```

Comment:
Looks like `licenses` is already an array in the result:

![screen shot 2015-04-08 at 1 56 12 am](https://cloud.githubusercontent.com/assets/81969/7039607/80c52502-dd92-11e4-946c-56314f170bc7.png)

So all we need is:

```javascript
var licenses = item.licenses || [];
```